pkg/executor/docker/executor_test.go

```diff
@@ -99,11 +100,10 @@ func (s *ExecutorTestSuite) containerHttpURL() *url.URL {
 	return url
 }
 
-func (s *ExecutorTestSuite) curlTask() model.JobSpecDocker {
-	return model.JobSpecDocker{
-		Image:      CurlDockerImage,
-		Entrypoint: []string{"curl", "--fail-with-body", s.containerHttpURL().JoinPath("hello.txt").String()},
-	}
+func (s *ExecutorTestSuite) curlTask() model.EngineSpec {
+	return model.NewDockerEngineBuilder(CurlDockerImage).
+		WithEntrypoint("curl", "--fail-with-body", s.containerHttpURL().JoinPath("hello.txt").String()).
+		Build()
```

Comment:
I missed when we introduced these constants `CurlDockerImage`, but there is a script that gets run when you call `make images` that is meant to grep the source code for images and download them, so that you can test offline. Does that still work and if not can we delete it?

Reply:
I believe the changes you are referring to landed here(?): https://github.com/bacalhau-project/bacalhau/pull/2558

Sorry, but I don't understand the ask here, is there something I need to change?

Reply:
`make images` (TIL) downloads the wrong version of curlimages/curl because of the way the constants are defined.
@simonwo when you're asking for it to be deleted, you mean the constants?

Reply:
No I mean the `make images` – I don't think it works now that we've got these constants. We don't have to do it in this PR.

Reply:
I am still not sure I understand what the problem is here, but if we are calling script that parses golang source code for docker images I think we might be doing something wrong. It will be hard to detect breakages (such as this?) with that pattern. @simonwo can you file any necessary issues for this work?